.devcontainer/README.md

```diff
@@ -0,0 +1,41 @@
+# `kube-rs` Development container
+
+This directory provides a [_devcontainer_][dc] configuration that configures a
+reproducible development environment for this project. This base image should
+contain only the bare necessities to get up and running with `kube-rs`.
+Customizations should be made in per-user configuration.
```

Comment:
Do you know if it's possible to add in tasks to this? As far as I can tell the customizable `Run Tasks` stuff would let us specify kube-rs specific test runners such as:

```json
		{
			"type": "cargo",
			"command": "test",
			"args": ["--lib", "--all"],
			"problemMatcher": [
				"$rustc"
			],
			"group": "test",
			"label": "kube-rs: unit tests"
		}
```

but not sure if this is embeddable

Reply:
This is probably configurable via devcontainer's `.settings`.

Reply:
I think the best way to handle this is to add a `.vscode/tasks.json` to version control. probably best for its own change

Reply:
ok, this isn't important if it's not part of the core flow. it also potentially creates yet another place to store information like 'what all the unit tests mean' (currently makefile + github actions).